web/src/narrow.js

```diff
@@ -422,6 +422,7 @@ export function activate(raw_operators, opts) {
     // most users aren't going to send a bunch of a out-of-narrow messages
     // and expect to visit a list of narrows, so let's get these out of the way.
     compose_banner.clear_message_sent_banners();
+    compose_banner.clear_unmute_topic_notifications();
```

Comment:
Hmm, why doesn't this count as just a variant of message-sent banners? I see you add this in the same places as those, and my read of the code is that this should qualify for that category, since it's a banner triggered by sending a message. (Most of the others are for validation issues before sending a message).

Reply:
OK; we can probably document that with comments and go from there. (I think in an ideal world we'd end up with an object describing each banner, and these would be send banners but not have the fade-on-scroll flag)

Reply:
I finally ended up adding a parameter `include_unmute_banner` to `clear_message_sent_banners` defaulting to true, in which case it calls `clear_unmute_topic_notifications`.

There are 3 cases where we need to pass this in as false, and a couple cases where we need to directly just call `clear_unmute_notifications`. This reduces a lot of the simultaneous calls to both functions, while also supporting the few cases where we need separate handling of these banners.

Reply:
Sounds reasonable! Posted a comment on one of those places.